config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -3639,3 +3639,66 @@ presubmits:
           defaultMode: 256
           secretName: ssh-security
     trigger: (?m)^/test( | .* )pull-security-kubernetes-verify,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: security
```

Comment:
Is this the same cluster used today for the jobs in sig-windows-config.yml? If not, some secrets need to be migrated over.

Reply:
We do not, as far as I know, have a mechanism to sync secrets between clusters, so you're probably right.

It's unclear to me what level of visibility this would bring to fixes under embargo, so I'm inclined to merge to unblock you, and leave this "broken" until we have had time to figure out what makes sense here.